students/student_framllat/session04/trigrams.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env python3
+
+import re
+import os
+import string
+
+"""
+trigrams
+"""
+
+# import file into a list
+with open('./four_sc.txt') as f:
+    #word_list = ()
+    word_list = []
+    lines = f.readlines()
+    for l in lines:
```

Comment:
you can do:
```
    for l in f:
```

and it will do the same thing, without loading it into memory all at once.